tests/4.5/taskloop/test_taskloop_final.c

```diff
@@ -0,0 +1,107 @@
+//--------------------------------------------- test_taskloop_final.c ----------------------------------------//
+//
+// OpenMP API Version 4.5 August 2015
+//
+// This test checks the taskloop directive with the if clause specified.
+// The 'taskloop' construct parallelize loops with independent iterations by creating tasks. 
+// It allows for efficient parallel execution of loop iterations by distributing them among multiple threads. 
+// The 'final clause is used to ensure that all tasks in taskloop are run by same thread if the 
+// final(expression) is TRUE, else different threads could execute the tasks.      
+//----------------------------------------------------------------------------------------------------------//
+
+#include <omp.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include "ompvv.h"
+
+#define NUM_TASKS 10
+#define NUM_THREADS 10
+
+#define M 100
+
+int thread_ids[NUM_TASKS][3]; //Array 2nd diemention value i.e 3 is depend on number of omp task under taskloop
```

Comment:
Change the comment as follows: 
The 2nd dimension of this array (i.e., 3) is dependent on the number of task constructs in the taskloop construct.

Reply:
updated